Portal/src/Datahub.Portal/Pages/Public/Login.razor

```diff
@@ -6,33 +6,82 @@
 @inject IUserInformationService _userInformationService
 @inject DatahubPortalConfiguration _portalConfiguration
 
+<style>
+    .custom-email-field {
+        padding: 4px; /* Adjust padding as needed */
+        width: 100%; /* Make the field take full width of its container */
+        max-width: none; /* Remove any max-width constraint */
+        box-sizing: border-box; /* Ensure padding is included in the width */
+    }
+
+    .custom-email-field:focus {
+        border-color: #007bff;
+        outline: none;
+    }
+    .flex-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+    }
+
+    .flex-item {
+        width: 100%;
+        max-width: 400px; 
+        text-align: center;
+    }
+</style>
+
+<EditForm Model="@loginModel">
+
 <PageTitle>@Localizer["Login - Federal Science DataHub"]</PageTitle>
 
 <MudStack Class="my-16 py-16">
     <MudStack AlignItems="AlignItems.Center" Class="mt-4">
-        <MudText Typo="Typo.h2" Class="text-center">@Localizer["Please login to continue"]</MudText>
+        <div class="flex-container">
+            <MudText Typo="Typo.h2" Class="text-center flex-item">@Localizer["Please login to continue"]</MudText>
+            <MudStack Class="form-group flex-item" AlignItems="AlignItems.Center">
+                    <MudTextField id="Email" T="string" Label="@Localizer["Email address"]" @bind-Value="loginModel.Email" Variant="Variant.Outlined" Class="custom-email-field my-2" />
+            </MudStack>
+        </div>
```

Comment:
Can we rework this to use `MudStack` attributes or direct styling instead of custom css classes

Reply:
Done